docs/tutorials/word-count/tutorial.md

```diff
@@ -202,8 +202,8 @@ In the end we would produce 5 messages in total, like so:
 
 ```python
 # two shown here...
-{kafka_key: "bob", kafka_value: ["bob", 1]}
-{kafka_key: "likes", kafka_value: ["likes", 2]}
+{kafka_key: "bob", kafka_value: ("bob", 1)}
+{kafka_key: "likes", kafka_value: ("likes", 2)}
```

Comment:
So it turns out the original is technically correct (though in reality it's `{kafka_key: b"bob", kafka_value: '["bob", 1]'}`, etc..., but I wrote it like this to make it a little more reader-friendly =) )

It actually converts the tuples into a list when it does `JSON` serialization, but I admit that's not super obvious behavior. Rather than changing this, we could instead mention that below it as like, a note.

Reply:
Ack. My bad.. this is how they were appearing on printing. But the values in Kafka show them with [].
I have reverted that change, and added a note instead.